src/lightning_fabric/connector.py

```diff
@@ -541,7 +541,7 @@ def _argument_from_env(name: str, current: Any, default: Any) -> Any:
         if env_value is not None and env_value != str(current) and str(current) != str(default):
             raise ValueError(
                 f"Your code has `Fabric({name}={current!r}, ...)` but it conflicts with the value "
-                f"`--{name}={current}` set through the CLI. "
+                f"`--{name}={env_value}` set through the CLI. "
```

Comment:
Wouldn't this be clearer?


```suggestion
                f"`{env_key}={env_value}` set through the environment. "
```

where `env_key = "LT_" + name.upper()`

Reply:
No, because we don't intend users to set the env variables manually (not documented anywhere). The env vars are just a means for us to enable the CLI functionality in `lightning run model ...`. So I would keep the error message focused on CLI usage.